### ISSUE TITLE
for `asap-alchemy gather`, ability to pull down separate replicates

### ISSUE THREAD

**Issue by JenkeScheen** [choderalab/asapdiscovery#645]
After discussions with @jthorton, it looks like we are pulling results down from `alchemiscale` in a consolidated way (and the confidence is the standard error of the prediction). For retrospective/debugging purposes, we also need to be able to check the individual replicates' predictions so that we can compare them.

**Comment by hmacdope** [comment]
@jthorton @JenkeScheen is this still an issue?

**Comment by JenkeScheen** [comment]
yes still an issue, currently we can only pull the mean of replicates
